core/templates/dev/head/pages/exploration_player/StatsReportingService.js

```diff
@@ -133,22 +133,22 @@ oppia.factory('StatsReportingService', [
           aggregatedStats.state_stats_mapping[stateName].first_hit_count += 1
 
           postStatsToBackend();
-        }
-
-        $http.post(getFullStatsUrl('EXPLORATION_STARTED'), {
-          params: params,
-          session_id: sessionId,
-          state_name: stateName,
-          version: explorationVersion
-        });
+        } else {
```

Comment:
I'm a bit confused -- why are we putting these into else blocks? Regardless of whether the stats framework is implemented, we should be recording and storing the raw events in the event logs -- right?

Reply:
Oh. My bad. For some reason, I was suddenly thinking that event models were still being recorded in the aggregated stats. Just checked that it didn't. Thanks for pointing that out